ui/layout-pages-fixed.md

```diff
@@ -0,0 +1,377 @@
+When enabled the navigation bar on top of a page does not get displayed on this page.
+In conjunction with hiding the sidebar via its pin icon, this can be used for full screen display, e.g. in a fullscreen browser or when used as Home Screen App on iOS.
+Defaults to false.
+
+_Hide Sidebar Icon:_
+With the navigation bar hidden, an icon is displayed on the top left corner when the sidebar is closed.
+Enabling this hides the icon. Defaults to false.
+
+_Show Fullscreen Icon:_
+Show a fullscreen icon on the top right corner to enter browser fullscreen mode.
+Defaults to false.
+
+### Fixed Canvas Layouts
+
+To define your screen size open the canvas menu ![configure-canvas.png](images%2Fconfigure-canvas.png) and click **Configure Canvas Layout**.
+
+The layout editor shows a gray surface as the _virtual screen area_, which represents your defined screen.
+You can now place widgets on that screen by clicking the **Add Widget** button.
+Just like with responsive layouts a placeholder will appear, allowing you to choose widgets from the library.
+
+Additionally, you can move the widget within the virtual screen area by dragging its upper left corner at the arrow icon shown.
+You can also resize the widget by grabbing the handles.
+Depending on the widget, you may also toggle the **Autosize** option in the widget options available from the gear icon on the top right corner of each widget to have the widget take its natural size.
+
+If you toggle the grid using the grid button, positioning and resizing will snap to the grid but untouched widgets will remain at their current size and position.
+The pitch of that grid can be adjusted in the layout configuration.
+
+The gear icon in the top right corner of each widget allows you to configure or remove a widget.
+
+#### Configuration
+
+The following parameters can be adjusted for a Fixed Canvas Layout:
+
+##### Layout Settings
+
+_Grid size:_
+Pitch of the grid when the editor grid button is enabled, in CSS pixels, to snap the content.
+
+##### Screen Settings
+
+_Screen Width:_
+Screen width in CSS pixels. Defaults to 1280.
+
+_Screen Height:_
+Screen height in CSS pixels. Defaults to 720.
+
+_Scaling:_
+Enabling this scales the defined screen to the width available in the browser window. All widgets are resized accordingly.
+While this works well in many cases, it can lead to unpredictable styling issues, especially on large divergence from the defined screen width.
+Defaults to false.
+
+_Image URL:_
+URL of the image to display in the background. The image is stretched to fit the screen size, while preserving its aspect ratio.
+If you want to achieve a different placement, adding margins to the image in an image editor will be necessary.
+
+This setting is in particular in important if you want to make your background image interactive, in which case it has to be of type SVG.
+See below for more information on [interactive backgrounds](layout-pages-fixed.html#interactive-backgrounds).
```

Comment:
Here the anchor link alone should be enough instead of having the filename as well:
#interactive-backgrounds

Reply:
see comment above